packages/components/file-uploader/src/file-uploader.css

```diff
@@ -0,0 +1,421 @@
+  animation-timing-function: linear;
+  background-color: var(--ts-color-blue);
+  border-radius: 50%;
+  content: '';
+  height: 22px;
+  left: 15px;
+  position: absolute;
+  top: 8px;
+  width: 22px;
+}
+
+:host > .file-card-wrapper > .file-card--download:hover .icon:first-of-type span:after {
+  animation-name: appear;
+  animation-duration: 0.15s;
+  animation-iteration-count: 1;
+  animation-timing-function: linear;
+  color: white;
+  content: '↓';
+  font-size: 17px;
+  left: var(--ts-unit);
+  position: absolute;
+  top: var(--ts-unit-half);
+}
+
+@keyframes appear {
+  0% {
+    top: 30px;
+  }
+  100% {
+    top: var(--ts-unit-half);
+  }
+}
+
+:host > .file-card-wrapper > .file-card--download label:hover {
+  cursor: pointer;
+}
+
+:host > .file-card-wrapper > .file-card--download small > small {
+  text-transform: uppercase;
+}
+
+:host > .file-card-wrapper > .file-card--download .filetype .message {
+  color: var(--ts-color-blue);
+  text-transform: unset;
+}
+
+:host > .file-card-wrapper > .file-card--download > .icon:last-of-type {
+  background-color: var(--ts-color-white);
+}
+
+:host > .file-card-wrapper > .file-card--download > .icon:last-of-type > .file-card--menu {
+  color: var(--ts-color-gray-dark);
+  display: none;
+  font-size: calc(3 * var(--ts-unit-half));
+  margin-top: -13px;
+}
+
+/* Dragging state.................................................. */
+
+:host > .file-upload-wrapper .draggable-info {
+  display: none;
+}
+
+:host > .file-upload-wrapper.dragover *  {
+  display: none;
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info * {
+  display: unset;
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info  {
+  display: flex;
+}
+
+:host > .file-upload-wrapper.dragover {
+  align-items: center;
+  background: #fff;
+  border-radius: 3px;
+  border: 2px solid #A5DFFF;
+  display: flex;
+  height: calc(6.5 * var(--ts-unit-half));
+  justify-content: center;
+  width: 300px;
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info {
+  color: #00B0FF;
+  font-weight: 600;
+  font-size: 14px;
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info .icon { 
+  align-items: center;
+  border-radius: 50%;
+  color: var(--ts-color-white);
+  display: flex;
+  justify-content: center;
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info .icon {
+  background-color: var(--ts-color-blue);
+  height: var(--ts-unit);
+  margin-right: calc(0.5 * var(--ts-unit-half));
+  width: var(--ts-unit);
+}
+
+:host > .file-upload-wrapper.dragover .draggable-info .icon > span {
+  font-size: 16px;
+  font-weight: bold;
+}
+
+/* Right-to-left support.................................................. */
+
+:host > .file-upload-wrapper.right-left > .file-upload-button {
+  flex-direction: row-reverse;
+}
+
+:host > .file-upload-wrapper.right-left > label {
+  flex-direction: row-reverse;
+  padding-right: calc(1px + var(--ts-unit-half));
+}
+
+:host > .file-upload-wrapper.right-left > label > svg {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Simplify this over-specified selector. Maximum allowed depth: 3. Actual depth: 4 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aoverspecific-selectors)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)